apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -282,14 +267,29 @@ export interface IConfigFile {
    */
   extends?: string;
 
+  /**
+   * The root folder for the project.  This folder typically contains the tsconfig.json and package.json
+   * config files.
+   *
+   * @remarks
+   *
+   * The `projectFolder` path is resolved relative to the folder containing api-extractor.json.
+   *
+   * The default value for `projectFolder` is the token `<lookup>`, which means the folder is determined by traversing
+   * parent folders, starting from the folder containing api-extractor.json, and stopping at the first folder
+   * that contains a tsconfig.json file.  If a tsconfig.json file cannot be found in this way, then an error
```

Comment:
Should this be `tsconfig.json` or `package.json`?

Reply:
My reasoning is that I would like package.json to eventually be optional, since e.g. Office Add-ins doesn't have one. The tsconfig.json is also optional, but it is more essential since it tells us how to interpret the typings.